lib/blockchain_double.js

```diff
@@ -305,28 +293,15 @@ BlockchainDouble.prototype.popBlock = function(callback) {
         var txHash = to.txHash(tx);
 
         requests.push(
-          self.database.transactions.del.bind(
-            self.database.transactions,
-            txHash
-          ),
-          self.database.transactionReceipts.del.bind(
-            self.database.transactionReceipts,
-            txHash
-          )
+          self.database.transactions.del.bind(self.database.transactions, txHash),
```

Comment:
I think these will need to be updated to reference `self.data.transactions` to make this merge-conflict free.